mobify/platform-scaffold
# PLP and Nav links go to canonical site

## Changes
- PLP links to original site
- Nav links to original site

## How to test-drive this PR
- Check out http://localhost:3000/books.html, http://localhost:3000/potions.html
- Tap on links in the menu, a product in the list
- Observe they take you to the right place on www.merlinspotions.com
